cluster-autoscaler/main.go

```diff
@@ -177,6 +178,7 @@ var (
 	clusterAPICloudConfigAuthoritative = flag.Bool("clusterapi-cloud-config-authoritative", false, "Treat the cloud-config flag authoritatively (do not fallback to using kubeconfig flag). ClusterAPI only")
 	cordonNodeBeforeTerminate          = flag.Bool("cordon-node-before-terminating", false, "Should CA cordon nodes before terminating during downscale process")
 	daemonSetEvictionForEmptyNodes     = flag.Bool("daemonset-eviction-for-empty-nodes", false, "DaemonSet pods will be gracefully terminated from empty nodes")
+	enableRefineUsingSimilarNodeGroups = flag.Bool("refine-using-similar-nodegroups", false, "Refine empty nodegoups capacities evaluations using real-world nodes from similar nodegroups when available.")
```

Comment:
I'm not sure how helpful that description is to end user who's not already very familiar with CA internals. What this flag does is pretty hard to explain without going into details, but maybe we can try to focus on how it impacts users?

Reply:
Updated following those guidelines (still not perfect, but I tried to remain concise)